1/test/run

```diff
@@ -22,13 +22,8 @@ function cleanup() {
     echo "Stopping and removing container $CONTAINER..."
     docker stop $CONTAINER
     exit_status=$(docker inspect -f '{{.State.ExitCode}}' $CONTAINER)
-    # jenkins "normally" exits with code 143:
-    # http://jenkins-ci.361315.n4.nabble.com/Jenkins-exit-code-143-due-to-Unexpected-termination-of-the-channel-td4652347.html
-    # tl;dr: 128+signal=return code, and "15" equates to SIGTERM
-    if [ "$exit_status" != "143" ]; then
-      echo "Dumping logs for $CONTAINER"
-      docker logs $CONTAINER
-    fi
+    echo "Dumping logs for $CONTAINER"
+    [[ "${exit_status}" != "0" ]] && docker logs $CONTAINER
```

Comment:
checking for exit status 0 is misleading and pointless though, because jenkins never exits with RC=0.

so either remove the check entirely, or put it back to 143 because "143" is the successful exit code for jenkins when it was sigterm'd, per the comment i had in there.